solr/solr-ref-guide/src/jwt-authentication-plugin.adoc

```diff
@@ -54,6 +54,8 @@ rolesClaim           ; What claim id to pull user roles from. The claim must the
 claimsMatch          ; JSON object of claims (key) that must match a regular expression (value). Example: `{ "foo" : "A|B" }` will require the `foo` claim to be either "A" or "B". ;
 adminUiScope         ; Define what scope is requested when logging in from Admin UI ; If not defined, the first scope from `scope` parameter is used
 redirectUris         ; Valid location(s) for redirect after external authentication. Takes a string or array of strings. Must be the base URL of Solr, e.g., https://solr1.example.com:8983/solr/ and must match the list of redirect URIs registered with the Identity Provider beforehand. ; Defaults to empty list, i.e., any node is assumed to be a valid redirect target.
+trustedCerts         ; One or more X.509 SSL certificates in plaintext PEM or PKCS#7 formats, that should be trusted when talking to IdPs. Newlines must be replaced with `\n`. ; Defaults to Java truststore
```

Comment:
I'd like to see more explanation when somebody would need this, similar to what was described in the JIRA, e.g. explain that Jetty's TrustStore is ignored if running with HTTP with an SSL-terminating proxy/LB/ingress in front.

Reply:
I added a separate paragraph explaining more, and linked to it from the table.